dbms/src/Debug/dbgFuncCoprocessor.cpp

```diff
@@ -994,10 +994,14 @@ std::tuple<QueryTasks, MakeResOutputStream> compileQuery(
         };
 
     /// finalize
-    std::unordered_set<String> used_columns;
-    for (auto & schema : root_executor->output_schema)
-        used_columns.emplace(schema.first);
-    root_executor->columnPrune(used_columns);
+    if (!context.isExecutorTest())
```

Comment:
Why this change?

Reply:
> Why this change?

Since in executor test, we don't call initOutputInfo() in DagContext.

Reply:
OK, can add some comments here